### DISCUSSION TITLE
Welcome to JSqlParser Discussions!

### DISCUSSION THREAD

**Discussion by wumpz** [JSQLParser/JSqlParser#1114] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jayant-kumar-yadav** [comment]
I am trying to parse "show indexes from  my_table" but it is throwing ParseException as:

net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "from" "FROM"
         at net.sf.jsqlparser.parser.CCJSqlParser.generateParseException(CCJSqlParser.java:31234)
	at net.sf.jsqlparser.parser.CCJSqlParser.jj_consume_token(CCJSqlParser.java:31067)
	at net.sf.jsqlparser.parser.CCJSqlParser.Statement(CCJSqlParser.java:163)

What is wrong here?

**Comment by manticore-projects** [comment]
Greetings.

Your syntax `show indexes from my_table` is not supported, since it does not apply to any SQL Standard or major RDBMS (as far as I know).

[Supported syntax are shown here.](https://www.manticore-projects.com/JSQLParser/syntax.html)

You may want to provide or sponsor an implementation of this unsupported feature.

**Comment by wumpz** [comment]
However, there are some types of `show` already implemented:

```sql
show mytable;
show columns from mytable;
```

So following the pass there should be problem.

PRs are welcome.
